### PR TITLE
Conv concat relu quantization

### PR DESCRIPTION
**Pull request by sfraczek**
Speeds up both fp32 and int8 execution on mkldnn

### CODE REVIEW

**Comment by luotao1** [comment]
> Speeds up both fp32 and int8 execution on mkldnn

Could you give some detail benchmark here? And which model will be benefited from this PR?

File: paddle/fluid/framework/ir/mkldnn/conv_concat_relu_mkldnn_fuse_pass_tester.cc [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+#include "paddle/fluid/framework/op_proto_maker.h"
+
+namespace paddle {
+namespace framework {
+namespace ir {
+
+void SetOp(ProgramDesc* prog, const std::string& type,
+           const std::vector<std::string>& inputs,
+           const std::vector<std::string>& outputs, bool use_mkldnn = true) {
+  // TODO(sfraczek): add code
```
Comment (wojtuss) [inline paddle/fluid/framework/ir/mkldnn/conv_concat_relu_mkldnn_fuse_pass_tester.cc]: Please, remove the comment

File: paddle/fluid/framework/ir/mkldnn/conv_concat_relu_mkldnn_fuse_pass_tester.cc [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+}
+
+void MainTest(const ProgramDesc& prog, bool fuse_relu) {
+  std::unique_ptr<ir::Graph> graph(new ir::Graph(prog));
+
+  auto pass = PassRegistry::Instance().Get("conv_concat_relu_mkldnn_fuse_pass");
+
+  int original_nodes_num = graph->Nodes().size();
+
+  graph.reset(pass->Apply(graph.release()));
```
Comment (wojtuss) [inline paddle/fluid/framework/ir/mkldnn/conv_concat_relu_mkldnn_fuse_pass_tester.cc:96]: Please, change the order of lines 96 and 98. Empty lines could also be reduced.

File: paddle/fluid/framework/ir/mkldnn/conv_concat_relu_mkldnn_fuse_pass_tester.cc (hunk 1 above)
Comment (sfraczek) [reply paddle/fluid/framework/ir/mkldnn/conv_concat_relu_mkldnn_fuse_pass_tester.cc]: ok

**Comment by wojtuss** [comment]
> > Speeds up both fp32 and int8 execution on mkldnn
> 
> Could you give some detail benchmark here? And which model will be benefited from this PR?

@luotao1 , GoogleNet will benefit from this fuse. Having fused `relu`s into `convolution`s improves FP32 performance on my developer machine by ~8%. For INT8 performance is >50% better and accuracy diff to FP32 is even smaller.

**Review by wojtuss — DISMISSED** [review]

**Review by luotao1 — DISMISSED** [review]
LGTM

**Review by wojtuss — APPROVED** [review]

**Pull request merged**